CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [11.0.0] - 2023-12-14
+
+### Changed
+- `measureme`: Update StringId and Addr sizes from u32 to u64 ([GH-216])
+- `analyzeme`: New support for v9 file format, which uses larger event sizes
```

Comment:
Please also add the link to the PR here.

Reply:
Updated